eng/Versions.props

```diff
@@ -7,7 +7,7 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>2</PatchVersion>
     <SdkBandVersion>7.0.100</SdkBandVersion>
-    <PackageVersionNet6>6.0.12</PackageVersionNet6>
+    <PackageVersionNet6>6.0.13</PackageVersionNet6>
```

Comment:
This value should probably be calculated off of the .NET 7 version, rather than manually updated.

Reply:
it needs to match the sdk downlevel version but we don't get the sdk update.

Reply:
yeah, that is definitely better than it getting missed